lib/include/scion/scion.h

```diff
@@ -42,149 +42,149 @@ enum scion_error {
 	/**
 	 * Something went wrong.
 	 */
-	SCION_GENERIC_ERR = -1,
+	SCION_ERR_GENERIC_ERR = -1,
 	/**
 	 * Memory allocation failed.
 	 */
-	SCION_MEM_ALLOC_FAIL = -2,
+	SCION_ERR_MEM_ALLOC_FAIL = -2,
 	/**
 	 * No paths to the destination were found.
 	 */
-	SCION_NO_PATHS = -3,
+	SCION_ERR_NO_PATHS = -3,
 	/**
 	 * The buffer provided was not large enough.
 	 */
-	SCION_BUFFER_SIZE_ERR = -4,
+	SCION_ERR_BUFFER_SIZE_ERR = -4,
```

Comment:
maybe rename this to `SCION_ERR_BUF_TOO_SMALL`

Reply:
done